sdk/ai/azure-ai-projects/samples/agents/async_samples/sample_agents_vector_store_enterprise_file_search_async.py

```diff
@@ -0,0 +1,97 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+"""
+FILE: sample_agents_vector_store_enterprise_file_search_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to add files to agent during the vector store creation.
+
+USAGE:
+    python sample_agents_vector_store_enterprise_file_search_async.py
+
+    Before running the sample:
+
+    pip install azure.ai.projects azure-identity azure-ai-ml
+
+    Set this environment variables with your own values:
+    PROJECT_CONNECTION_STRING - the Azure AI Project connection string, as found in your AI Studio Project.
+"""
+import asyncio
+import os
+from azure.ai.ml import MLClient
+from azure.ai.ml.constants import AssetTypes
+from azure.ai.ml.entities import Data
+
+from azure.ai.projects.aio import AIProjectClient
+from azure.ai.projects.models import (
+    FileSearchTool,
+    VectorStoreDataSource,
+    VectorStoreDataSourceAssetType
+)
+from azure.identity import DefaultAzureCredential
+
+
+async def main():
+    # Create an Azure AI Client from a connection string, copied from your AI Studio project.
+    # At the moment, it should be in the format "<HostName>;<AzureSubscriptionId>;<ResourceGroup>;<HubName>"
+    # Customer needs to login to Azure subscription via Azure CLI and set the environment variables
+    credential = DefaultAzureCredential()
+    project_client = AIProjectClient.from_connection_string(
+        credential=credential, conn_str=os.environ["PROJECT_CONNECTION_STRING"]
+    )
+
+    async with project_client:
+
+        ml_client = MLClient.from_config(credential)
```

Comment:
Could we use also upload_file method here from projects?

Reply:
Fixed